spec/lib/models/node-spec.js

```diff
@@ -152,5 +152,72 @@ describe('Models.Node', function () {
             });
         });
 
+        describe('tags', function() {
+            before(function() {
+                this.subject = this.attributes.tags;
+            });
+
+            it('should be an array', function() {
+                expect(this.subject.type).to.equal('array');
+            });
+
+            it('should default to empty', function() {
+                expect(this.subject.defaultsTo).to.deep.equal([]);
+            });
+        });
+
+        describe('Node Tag Functions', function() {
+            var collection = {
+                update: sinon.stub().resolves()
+            };
+
+            before(function() {
+                sinon.stub(this.model, "native", function(cb) { cb(null, collection); });
```

Comment:
No qualms with the tactics, just trying to grok where `.native` normally resides. I'm not clear on where it's getting promisified. Is native something that's already in the waterline model objects and is a convenient hook for capturing the flow that would otherwise go down into the MongoDB drivers?